token-utxo/README.md

```diff
@@ -0,0 +1,205 @@
+
+## Deploy the smart contract to the channel
+
+You can use the test network script to deploy the UTXO token contract to the channel that was just created. Deploy the smart contract to `mychannel` using the following command:
+```
+./network.sh deployCC -ccn token_utxo -ccp ../token-utxo/chaincode-go/
+```
+
+The above command deploys the go chaincode with short name `token_utxo`. The smart contract will utilize the default endorsement policy of majority of channel members.
+Since the channel has two members, this implies that we'll need to get peer endorsements from 2 out of the 2 channel members.
+
+Now you are ready to call the deployed smart contract via peer CLI calls. But let's first create the client identities for our scenario.
+
+## Register identities
+
+The smart contract supports UTXO ownership based on individual client identities from organizations that are members of the channel. In our scenario, the minter of the tokens will be a member of Org1, while the recipient will belong to Org2. To highlight the connection between the `GetClientIdentity().GetID()` API and the information within a user's certificate, we will register two new identities using the Org1 and Org2 Certificate Authorities (CA's), and then use the CA's to generate each identity's certificate and private key.
+
+First, we need to set the following environment variables to use the the Fabric CA client (and subsequent commands)
+```
+export PATH=${PWD}/../bin:${PWD}:$PATH
+export FABRIC_CFG_PATH=$PWD/../config/
+```
+
+We will use the Org1 CA to create the minter identity. Set the Fabric CA client home to the MSP of the Org1 CA admin (this identity was generated by the test network script):
+```
+export FABRIC_CA_CLIENT_HOME=${PWD}/organizations/peerOrganizations/org1.example.com/
+```
+
+You can register a new minter client identity using the `fabric-ca-client` tool:
+```
+fabric-ca-client register --caname ca-org1 --id.name minter --id.secret minterpw --id.type client --tls.certfiles ${PWD}/organizations/fabric-ca/org1/tls-cert.pem
+```
+
+You can now generate the identity certificates and MSP folder by providing the enroll name and secret to the enroll command:
+```
+fabric-ca-client enroll -u https://minter:minterpw@localhost:7054 --caname ca-org1 -M ${PWD}/organizations/peerOrganizations/org1.example.com/users/minter@org1.example.com/msp --tls.certfiles ${PWD}/organizations/fabric-ca/org1/tls-cert.pem
+```
+
+Run the command below to copy the Node OU configuration file into the minter identity MSP folder.
+```
+cp ${PWD}/organizations/peerOrganizations/org1.example.com/msp/config.yaml ${PWD}/organizations/peerOrganizations/org1.example.com/users/minter@org1.example.com/msp/config.yaml
+```
+
+We can now use the Org2 CA to create the Org2 recipient identity. Set the Fabric CA client home the Org2 CA admin:
```

Comment:
Same comment as for the token-based account sample, I think it would make sense to introduce and use the org2 terminal here.

Reply:
done.